src/sdk-gen-aws.ts

```diff
@@ -0,0 +1,6 @@
+import * as AWS from "aws-sdk";
+
+//
+export const aws = AWS;
```

Comment:
I grab the type of this Identifier in the gen script and use it to parse out all the types.